src/js/block.js

```diff
@@ -0,0 +1,19 @@
+export default class Block {
+  constructor({ x, y, unitSize, color }) {
+    this.x = x;
+    this.y = y;
+    this.unitSize = unitSize;
+    this.color = color;
+  }
+
+  draw(container) {
+    this.div = document.createElement('div');
+    this.div.classList.add('block');
+    this.div.classList.add(`block_color_${this.color}`);
+    this.div.style.top = `${this.y * this.unitSize}px`;
+    this.div.style.left = `${this.x * this.unitSize}px`;
+    this.div.style.width = `${this.unitSize}px`;
+    this.div.style.height = `${this.unitSize}px`;
+    container.append(this.div);
```

Comment:
I don't think passing the container to this method is a good idea. The container should handle adding the blocks to the grid, and this method can return the html element (and renamed to `getHtmlElement`). But it's fine don't change it.